### PR TITLE
Add support for floor() and ceil()

### PR DESCRIPTION
**Pull request by tirkarp**
https://gitlab.com/purdue-informatics/variate/variate-solution-engine/-/issues/61

### CODE REVIEW

**Comment by tirkarp** [comment]
floor() and ceil() functions can be parsed in 3 different forms here:
1. `\floor(x)` and `\ceil(x)`
2. `\operatorname{floor}(x)` and `\operatorname{ceil}(x)`
3. `\lfloor x\rfloor` and `\lceil x\rceil`

File: latex2sympy.py [hunk 1]
```diff
@@ -814,6 +827,24 @@ def apply_nested(f, lst):
     return sympy.UnevaluatedExpr(result)
 
 
+def handle_floor(expr):
+    """
+    Apply floor() then return the evaluated expression.
```
Comment (builtbywill) [inline latex2sympy.py:832]: @tirkarp since we are using `evaluate=False` these methods are not technically returning evaluated expressions

Comment (tirkarp) [reply latex2sympy.py:826]: okay. wrong terminology.

**Comment by tirkarp** [comment]
> Looks good. Was this meant to target the gcd-lcm branch or develop?

gcd-lcm just because it's easier to merge changes.

**Pull request merged**